packages/synapse-react-client/src/style/components/_cards.scss

```diff
@@ -58,8 +58,13 @@ $text-color-lighter: #898989;
     @media (max-width: map-get(SRC.$breakpoints, 'medium')) {
       padding-top: 89px;
     }
-    .SRC-type,
+    .SRC-type {
+      color: inherit;
+    }
     .SRC-cardThumbnail {
```

Comment:
I noticed that in all of the designed, the icon is centered vertically when not on a details page (when it's not a header card)